fortran-inference/inference_scaling_imagenet.F90

```diff
@@ -82,52 +95,6 @@ subroutine init_client( client, rank, use_cluster, timing_unit )
 
 end subroutine init_client
 
-subroutine set_model(client, device_type, num_devices, batch_size)
```

Comment:
Why did you remove set_model()? Are you always setting up the model via SmartSim before running the scaling study?

Reply:
yes!